config.json

```diff
@@ -2,5 +2,6 @@
     "log_level": "ERROR",
     "database_name": "birana",
     "database_uri": "mongodb://teachertian:aGMpG4BW8hQU9np@ds331135.mlab.com:31135/birana",
-    "sentry_dsn": "https://3cf3191bbc8343f78d32edf5545d4226@sentry.io/1390859"
+    "sentry_dsn": "https://3cf3191bbc8343f78d32edf5545d4226@sentry.io/1390859",
+    "database_collection": "house_locations"
```

Comment:
I think we should use a more specific key name here, such as geometry_collection or house_location_collection etc.

Reply:
Agree!